<testsuites>
    <testsuite name="firstprotractortest;subtraction" tests="5" failures="0" errors="0" skipped="0" timestamp="Fri, 25 Nov 2016 21:42:45 GMT" time="5.993068855000001">
        <testcase name="Before undefined" classname="firstprotractortest;subtraction" time="0.000332816">
        </testcase>
        <testcase name="Given I open &quot;https://juliemr.github.io/protractor-demo/&quot;" classname="firstprotractortest;subtraction" time="0.664513823">
        </testcase>
        <testcase name="When I want to do &quot;5&quot; &quot;-&quot; &quot;2&quot;" classname="firstprotractortest;subtraction" time="0.265842064">
        </testcase>
        <testcase name="Then result of operation should equal &quot;3&quot;" classname="firstprotractortest;subtraction" time="5.062230966">
        </testcase>
        <testcase name="After undefined" classname="firstprotractortest;subtraction" time="0.000149186">
        </testcase>
    </testsuite>
    <testsuite name="firstprotractortest;newadditiontest" tests="5" failures="1" errors="0" skipped="0" timestamp="Fri, 25 Nov 2016 21:42:45 GMT" time="5.46913509">
        <testcase name="Before undefined" classname="firstprotractortest;newadditiontest" time="0.000085324">
        </testcase>
        <testcase name="Given I open &quot;https://juliemr.github.io/protractor-demo/&quot;" classname="firstprotractortest;newadditiontest" time="0.182664787">
        </testcase>
        <testcase name="When I want to do &quot;1&quot; &quot;+&quot; &quot;4&quot;" classname="firstprotractortest;newadditiontest" time="0.217677705">
        </testcase>
        <testcase name="Then result of operation should equal &quot;58&quot;" classname="firstprotractortest;newadditiontest" time="5.06862185">
            <failure message="AssertionError: expected &apos;5&apos; to equal &apos;58&apos;">AssertionError: expected &apos;5&apos; to equal &apos;58&apos;
    at /Users/singhs10/project_secondProTravisCucumberJS/features/step_definitions/actionwords.js:21:30
    at /Users/singhs10/.nvm/versions/node/v5.3.0/lib/node_modules/protractor/lib/element.js:665:16
    at [object Object].promise.Promise.goog.defineClass.invokeCallback_ (/Users/singhs10/.nvm/versions/node/v5.3.0/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:1337:14)
    at [object Object].promise.ControlFlow.goog.defineClass.goog.defineClass.abort_.error.executeNext_.execute_ (/Users/singhs10/.nvm/versions/node/v5.3.0/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:2776:14)
    at [object Object].promise.ControlFlow.goog.defineClass.goog.defineClass.abort_.error.executeNext_ (/Users/singhs10/.nvm/versions/node/v5.3.0/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:2758:21)
    at goog.async.run.processWorkQueue (/Users/singhs10/.nvm/versions/node/v5.3.0/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/goog/async/run.js:124:15)
    at process._tickCallback (node.js:382:9)</failure>
        </testcase>
        <testcase name="After undefined" classname="firstprotractortest;newadditiontest" time="0.000085424">
        </testcase>
    </testsuite>
</testsuites>